test_common/helpers/setup.rb

```diff
@@ -310,16 +315,18 @@ def answer_all_questions # rubocop:disable Metrics/MethodLength
 
       # Stub Calls to API
       def stub_et_api # rubocop:disable Metrics/MethodLength
-        stub_request(:post, "https://et-api-example.com/v2/respondents/response").
+        stub_request(:post, "https://et-api-example.com/v2/respondents/build_response").
           with(headers: { content_type: 'application/json', 'Accept': 'application/json' }).
           to_return(
             headers: { content_type: 'application/json' },
             body:
               {
-                "data": {
-                  "reference": "992000000100",
-                  "submitted_at": "2018-01-13 14:00",
-                  "pdf": "s3/link/to/form/pdf"
+                "meta": {
+                  "BuildResponse": {
+                    "reference": "992000000100",
+                    "submitted_at": "2018-01-13 14:00",
+                    "pdf": "s3/link/to/form/pdf"
```

Comment:
You won't get the pdf at this point from the API - as it will be generated asynchronously (as its slow).  We'll go through how to get this when you get back as I may need to experiment a little